extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -614,8 +616,12 @@ void resetInternal(DataSourceMetadata dataSourceMetadata)
           }
         }
         if (metadataUpdateSuccess) {
-          killTaskGroupForPartitions(JavaCompatUtils.keySet(resetKafkaMetadata.getKafkaPartitions()
-                                                                              .getPartitionOffsetMap()));
+          killTaskGroupForPartitions(
+              JavaCompatUtils.keySet(
```

Comment:
@gianm should I remove the class altogether? We don't ever build Druid against Java 7 anymore, correct?